### PR TITLE
Design updates

### PR DESCRIPTION
**Pull request by shishir0019**

### CODE REVIEW

**Comment by ardeay** [comment]
The space between the search input and the dropdown should be minimized 
image


like this 
image

**Comment by ardeay** [comment]
This space issues shows up in sealed too, since that component is used in many places 
image

**Comment by shishir0019** [comment]
image

File: pages/inventory/sealed.vue [hunk 1]
```diff
@@ -128,7 +128,7 @@ export default {
       this.search = q
     },
     deleteItem(id) {
-      fetch(`${api_url}inventory/remove/inventory_id=${id}`).then(
+      fetch(`${process.env.API_DOMAIN}/inventory/remove/inventory_id=${id}`).then(
```
Comment (ardeay) [inline pages/inventory/sealed.vue:131]: fetches need `auth` added to them

File: pages/inventory/sealed.vue [hunk 2]
```diff
@@ -141,7 +141,7 @@ export default {
         quantity: 1,
         language: 'EN',
       }
-      var apiURL = api_url + 'inventory/add/'
+      var apiURL = `${process.env.API_DOMAIN}/inventory/add/`
```
Comment (ardeay) [inline pages/inventory/sealed.vue:144]: fetches need `auth` added to them

**Review by ardeay — CHANGES_REQUESTED** [review]

File: pages/inventory/sealed.vue [hunk 3]
```diff
@@ -128,28 +128,29 @@ export default {
       this.search = q
     },
     deleteItem(id) {
-      fetch(`${api_url}inventory/remove/inventory_id=${id}`).then(
+      let token = this.$cookies.get('token');
+      fetch(`${process.env.API_DOMAIN}/inventory/remove/inventory_id=${id}?auth=${token}`).then(
```
Comment (ardeay) [inline pages/inventory/sealed.vue:132]: `d=${id}?auth=${token}` needs to be `d=${id}&auth=${token}` the question mark for the ampersand

**Review by ardeay — CHANGES_REQUESTED** [review]

**Pull request merged**